packages/installer/cos.sh

```diff
@@ -588,45 +740,38 @@ find_recovery() {
 # cos-upgrade-image: system/cos
 find_upgrade_channel() {
 
-    load_config
```

Comment:
mmh too bad we don't have tests for suc-upgrade. I remember we loaded config here because otherwise the config file would override what was the user provided flags from CLI, breaking it. https://github.com/rancher-sandbox/cOS-toolkit/pull/543/files#r693834604

Reply:
isnt this then fixed with the new order load?

Reply:
I think it is indeed, currently having a closer look :eyes: